Ex1/ex1.js

```diff
@@ -0,0 +1,11 @@
+//Scrieți o funcție care returnează pătratul unui număr
+//Funcția findSquare(num) returnează pătratul numărului primit ca date de intrare (este transmis ca argument în apelul funcției).
+
+
+function findSquare(num) {
+    return num * num;
+}
+
+console.log(findSquare(6));    //  36
```

Comment:
Se recomanda ca comentariile sa fie mereu intr-o linie noua, deci mai sus de linie decat in acelasi rand cu linia.